src/gui/editorwidgets/qgsrelationreferencewidget.h

```diff
@@ -269,6 +268,9 @@ class GUI_EXPORT QgsRelationReferenceWidget : public QWidget
     //! unset the currently related feature
     void deleteForeignKeys();
 
+    //! trigger save of the embedded referenced attribute form
+    bool saveReferencedAttributeForm();
```

Comment:
```suggestion
    /**
      * Trigger save of the embedded referenced attribute form.
      *
      * \since QGIS 3.44
      */
    bool saveReferencedAttributeForm();
```